schema.json

```diff
@@ -1,255 +1,266 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter Metadata Language Schema, Informative",
+  "description": "The (informative) schema for the Parameter Metadata Language
+                  from E1.37-5, Section 5. This schema is subject to change.",
   "definitions": {
-    "bitfield": {
+    "bitType": {
+      "description": "Describes the meaning of one bit in a bit field.",
       "type": "object",
       "properties": {
+        "name": { "$ref": "#/definitions/nameType" },
         "index": {
           "type": "integer",
           "minimum": 0
-        },
-        "name": {
-          "$ref": "#/definitions/name"
         }
       },
-      "required": ["index", "name"]
+      "required": [ "name", "index" ]
+    },
+    "bitFieldType": {
+      "description": "A bit field, a collection of 'bit' items.",
+      "properties": {
+        "name": { "$ref": "#/definitions/nameType" },
+        "type": { "const": "bitField" },
```

Comment:
I don't think JSON Schema Linter likes us overriding type here, I think I agree, I'm not sure if we need this type for our use (to differentiate actual objects), but if we do we should give it a different name. We may also need to tell it that the type of our const is "string" in JS land.

Reply:
"type" seems to validate okay in the v7 validators?